config/default/translations/messages-en.properties

```
@@ -131,4 +131,428 @@ task.pnc.danger_sign_followup_mother.title = PNC danger sign follow up - mother
+report.pregnancy.gestational_age.method_approx.lmp_approx_months = Months
+report.pregnancy.gestational_age.method_edd = EDD
+report.pregnancy.gestational_age.method_edd.u_edd = Date of Delivery
+
+report.pregnancy.gestational_age.method_lmp_summary = LMP Summarry
+
+report.pregnancy.gestational_age.g_lmp_date_8601 = LMP 8601
+report.pregnancy.gestational_age.g_lmp_date = LMP Date
+report.pregnancy.gestational_age.g_edd_8601 = EDD 8601
+report.pregnancy.gestational_age.g_edd = EDD
+report.pregnancy.gestational_age.method_none = Method None
+report.pregnancy.gestational_age.method_none.no_info_pregnancy_reason = Reason for registering patients
+
+report.pregnancy.anc_visits_hf = ANC Visits to Facility
+report.pregnancy.anc_visits_hf.anc_visits_hf_past = ANC Visits at Health Facility (Past)
+report.pregnancy.anc_visits_hf.anc_visits_hf_past.visited_hf_count = Number of times the patient visited the health facility for ANC 
+report.pregnancy.anc_visits_hf.anc_visits_hf_past.visited_dates_group =  ANC visit dates
+
+report.pregnancy.anc_visits_hf.anc_visits_hf_past.visited_dates_group.visited_date_ask_single = Date 
+report.pregnancy.anc_visits_hf.anc_visits_hf_past.visited_dates_group.visited_date_single = Date
+report.pregnancy.anc_visits_hf.anc_visits_hf_past.visited_dates_group.visited_dates_multiple_note = Visits Date
+report.pregnancy.anc_visits_hf.anc_visits_hf_past.visited_dates_group.visited_dates_count = Number of ANC visits
+report.pregnancy.anc_visits_hf.anc_visits_hf_past.visited_dates_group.visited_dates = Visit Dates
+report.pregnancy.anc_visits_hf.anc_visits_hf_past.visited_dates_group.visited_dates.0 = First Visit
+report.pregnancy.anc_visits_hf.anc_visits_hf_past.visited_dates_group.visited_dates.0.visited_date_ask_multiple =Visit
+report.pregnancy.anc_visits_hf.anc_visits_hf_past.visited_dates_group.visited_dates.0.visited_date = Date
+report.pregnancy.anc_visits_hf.anc_visits_hf_past.visited_dates_group.visited_dates.1 = Second Visit
+report.pregnancy.anc_visits_hf.anc_visits_hf_past.visited_dates_group.visited_dates.1.visited_date_ask_multiple =Visit
+report.pregnancy.anc_visits_hf.anc_visits_hf_past.visited_dates_group.visited_dates.1.visited_date = Date
+report.pregnancy.anc_visits_hf.anc_visits_hf_past.visited_dates_group.visited_dates.2 = Third Visit
+report.pregnancy.anc_visits_hf.anc_visits_hf_past.visited_dates_group.visited_dates.2.visited_date_ask_multiple =Visit
+report.pregnancy.anc_visits_hf.anc_visits_hf_past.visited_dates_group.visited_dates.2.visited_date = Date
+report.pregnancy.anc_visits_hf.anc_visits_hf_past.visited_dates_group.visited_dates.3 = Fourth Visit
+report.pregnancy.anc_visits_hf.anc_visits_hf_past.visited_dates_group.visited_dates.3.visited_date_ask_multiple =Visit
+report.pregnancy.anc_visits_hf.anc_visits_hf_past.visited_dates_group.visited_dates.3.visited_date = Date
+report.pregnancy.anc_visits_hf.anc_visits_hf_next = ANC Visits at Health Facility (Scheduled)
```

Should also be sentence case?

```suggestion
report.pregnancy.anc_visits_hf.anc_visits_hf_next = Scheduled ANC Health Facility Visits
```

---

Title case is appropriate since this is a new group header